str_analysis/data/known_pathogenic_variant_catalog__GRCh37__58_loci_with_offtargets.json

```diff
@@ -0,0 +1,2433 @@
+[
+   {
+      "LocusId": "AFF2",
+      "LocusStructure": "(GCC)*",
+      "ReferenceRegion": "chrX:147582151-147582211",
+      "VariantType": "RareRepeat",
```

Comment:
what are the options for this field? it seems like there are only "RareRepeat"s or "Repeat"s?